compiler/modulegraphs.nim

```diff
@@ -53,6 +53,10 @@ type
     concreteTypes*: seq[FullId]
     inst*: PInstantiation
 
+  SymInfoPair* = ref object
```

Comment:
Why is this a `ref object`? It should be just `object` as far as I can tell from how it's used.

Reply:
Fixed.